docs/LINUX.md

```diff
@@ -101,25 +32,59 @@ enabled=1
 gpgcheck=1
 gpgkey=https://assets.logdna.com/logdna.gpg" | sudo tee /etc/yum.repos.d/logdna.repo
 
-# Update the package
-sudo yum update logdna-agent
-```
-
-The agent will uninstall the previous version and reuse the existing configuration file, by default
-located in `/etc/logdna.conf`.
-
-If you defined a `logdna-agent.conf` configuration file on a different location, and want to use it instead, you
-can specify it on your systemd unit file:
-
-```unit file (systemd)
-[Service]
-Environment="LOGDNA_CONFIG_FILE=/your/path/to/logdna.conf"
-```
-
-"Next, restart the agent:
+# install the package via yum (skip this step if already installed)
+sudo yum -y install logdna-agent
 
-```shell script
-sudo systemctl restart logdna-agent
+# update the package via yum
+sudo yum update logdna-agent
 ```
-
-[ingestion-key]: https://docs.logdna.com/docs/ingestion-key
+#
+## Usage
+The agent uses [systemd](https://systemd.io/) to run as a Linux daemon. The installed package provides a `systemd` unit file for the `logdna-agent` service, which is defined to execute a daemon process with the compiled agent binary. The process that is started in the service is managed by `systemd` and can be interfaced with using `systemctl`.
+
+---
+### _Enabling the agent_
+- Reload the `systemd` unit files to ensure that the most recent version of the agent is used
```

Comment:
Let's rework this section so that it looks like this:

- Reload the `systemd` unit files to ensure that the most recent version of the agent is used.

  ```bash
  sudo systemctl daemon-reload

- Enable the `logdna-agent` service using the `systemctl` command.

  ```bash
  sudo systemctl enable logdna-agent